web/src/lightbox.js

```diff
@@ -272,7 +305,7 @@ function display_video(payload) {
     $iframe.attr("frameborder", 0);
     $iframe.attr("allowfullscreen", true);
 
-    $("#lightbox_overlay .player-container").html($iframe).show();
+    $("#lightbox_overlay .player-container").html($iframe);
```

Comment:
Why is it correct to remove the `.show` here?

Reply:
Since there is already a `$(".player-container").show();` at the start of this function and for videos, we don't use the `.player-container`, so it needs to stay hidden if it was hidden due to payload being a video.